cellprofiler/preferences.py

```diff
@@ -228,7 +227,7 @@ def config_write(key, value):
         shutup = wx.LogNull()
     __cached_values[key] = value
     if value is not None:
-        value = utf16encode(value)
+        value = value.encode('utf-16')
```

Comment:
Could this be replaced with the corresponding Python method?

```Python
value = value.decode("utf-16")
```

Reply:
I think this is supposed to be encoded so that it's written out in byte format here...although I may be swapping encode/decode